README.md

```diff
@@ -1,2 +1,143 @@
-# hello_world
-Repo created for excercise "Set up a 'Hello Microverse' project" of Microverse program, day 2
+    <a name="readme-top"></a>
+    
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] This link is broken, if you don't have any logo at the moment kindly remove it so as to have a professional README file. ℹ️

Reply:
The logo was removed, as requested.